gitlab-pipeline/stage/init.yml

```diff
@@ -16,7 +16,8 @@ init_workspace:
 
     - export WORKSPACE=$(realpath ${CI_PROJECT_DIR}/..)
     - apk --update add git openssh bash python3 curl py3-pip jq
-    - pip3 install --upgrade pyyaml==5.3.1
+    - wget https://raw.githubusercontent.com/mendersoftware/integration/b4f9871e578fee102c6e316e19cc3a602d2a70ed/extra/requirements.txt
+    - pip3 install -r requirements.txt
```

Comment:
By looking at it, I cannot tell why the release tool is required in the init job. I suspect it is a leftover.

Either update it as the rest, or try removing it. If we remove it, let's do it in a separate commit in case we have to urgently revert. But again, I don't see where it can possibly be used.